src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/DateTimeValue_TimeZone_Seattle.txt

```diff
@@ -0,0 +1,8 @@
+﻿// (UTC-08:00) Pacific Time (US & Canada)
+#SETUP: TimeZoneInfo(Pacific Standard Time)
+
+>> Text(DateAdd(DateTime(2022,3,13,1,0,0), 2, "hours"))
```

Comment:
>>> [](http://example.com/codeflow?start=0&length=2)

We don't have to repeat the same comment in paris file, but still good to call out "daylight savings time issue, see same expression in Paris .txt "